src/ifcblenderexport/blenderbim/bim/ui.py

```diff
@@ -917,6 +959,56 @@ def draw(self, context):
             row.operator("bim.set_north_offset")
 
 
+class BIM_PT_presentation_layers(Panel):
+    bl_label = "IFC Presentation Layers"
+    bl_idname = "BIM_PT_presentation_layer"
+    bl_options = {"DEFAULT_CLOSED"}
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "scene"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        props = context.scene.BIMProperties
+
+        layout.row().prop(props, "presentation_layers")
```

Comment:
I removed this line simply because this convention isn't shown anywhere else in the line